README.md

```diff
@@ -42,6 +42,7 @@ Follow instructions in https://kafka.apache.org/quickstart
     ./gradlew test  # runs both unit and integration tests
     ./gradlew unitTest
     ./gradlew integrationTest
+    ./gradlew test -Pkafka.test.run.flaky=true
```

Comment:
could you please add comment for it? for example, "This command is used to run the tests with the flaky tag."